book3/14-objects.mkd

```diff
@@ -234,55 +235,55 @@ an = PartyAnimal()
+*instancia* de la clase 'PartyAnimal'. Se ve como si fuera una llamada de función
+a la clase misma. Python construye el objeto con los datos y métodos adecuados,
+asignándolo luego a la variable 'an'. En cierto modo, esto es muy similar
+a la siguiente línea, que hemos estado usando todo este tiempo:
 
 ~~~~ {.python}
 counts = dict()
 ~~~~
 
-Here we instruct Python to construct an object using the `dict` template
-(already present in Python), return the instance of dictionary,
-and assign it to the variable `counts`.
+Aquí le ordenamos a Python construir un objeto usando la plantilla 'dict'
+(que viene incluida en Python), devolver la instancia del diccionario,
+y asignarla a la variable 'counts'.
 
-When the `PartyAnimal` class is used to construct an object, the variable `an`
-is used to point to that object.  We use `an` to access the code and data for that
-particular instance of the `PartyAnimal` class.
+Cuando se usa la clase 'PartyAnimal' para construir un objeto, la variable 'an'
+se usa para señalar ese objeto. Usamos 'an' para acceder al código y datos de esa
+instancia específica de la clase 'PartyAnimal'.
 
-Each Partyanimal object/instance contains within it a variable `x`
-and a method/function named `party`. We call the `party` method in this line:
+Cada objeto o instancia de PartyAnimal contiene una variable 'x'
+y un método/ función llamado 'party'. Llamamos al método 'party' en esta línea:
 
 ~~~~ {.python}
 an.party()
 ~~~~
 
-When the `party` method is called, the first parameter (which we call by convention `self`)
-points to the particular instance of the PartyAnimal object that `party` is called from.
-Within the `party` method, we see the line:
+Al llamar al método 'party', el primer parámetro (al que por convención llamamos 'self')
+apunta a la instancia específica del objeto PartyAnimal desde el que se llama a 'party'.
+Dentro del método 'party', vemos la siguiente línea:
 
 ~~~~ {.python}
 self.x = self.x + 1
 ~~~~
 
-This syntax using the *dot* operator is saying 'the x within self.' Each time `party()` is
-called, the internal `x` value is incremented by 1 and the value is printed out.
+Esta sintaxis utiliza el operador de *punto*, con lo que significa 'la x dentro de self'.
+Cada vez que se llama a 'party()', el valor interno de 'x' se incrementa en 1 y se imprime
+su valor.
 
-The following line is another way to call the `party` method within the `an`
-object:
+La siguiente línea muestra otra manera de llamar al método 'party' dentro del 
+objeto 'an':
 
 ~~~~ {.python}
 PartyAnimal.party(an)
 ~~~~
 
-In this variation, we access the code from within the class and explicitly
-pass the object pointer `an` as the first parameter (i.e., `self` within the method).
-You can think of `an.party()` as shorthand for the above line.
+En esta variante, accedemos al código desde el interior de la clase y explícitamente
+pasamos el objeto 'an' como el primer parámetro (es decir, 'self' dentro del método).
```

Comment:
Update to "pasamos el apuntador del objeto \`an\`"

Reply:
acabo de terminar de efectuar las correcciones mencionadas aquí. Usé buscar y reemplazar para cambiar los '' por backticks en todo el documento.